kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/importer/DataRecordMerger.java

```diff
@@ -119,21 +119,22 @@ private void mergeUpdate(final DataRecord dataRecord, final Map<DataRecord.Key,
     }
     
     private void mergeDelete(final DataRecord dataRecord, final Map<DataRecord.Key, DataRecord> dataRecords) {
-        DataRecord beforeDataRecord = dataRecords.get(dataRecord.getKey());
+        DataRecord beforeDataRecord = dataRecords.get(dataRecord.getOldKey());
```

Comment:
1, `dataRecord.getOldKey()` might be `null`?

2, Could `dataRecord.getOldKey()` method name be updated?

Reply:
Delete record before have value, after is nul, the getOldKey() will get the before uniquke value, is't not null

Reply:
OK